Wasm bindgen code does not compile if i include "use std::borrow::BorrowMut;"

Structs containing wasm_bindgen attribute stop compiling if BorrowMut is included 
```
use std::borrow::BorrowMut;
```

Receive the following error pointing to #[wasm_bindgen]

error[E0283]: type annotations required: cannot resolve `std::boxed::Box<wasm_bindgen::__rt::WasmRefCell<application::Mystruct>>: std::borrow::BorrowMut<_>`

Struct in the example was pretty simple 

```
#[wasm_bindgen]
struct MyStruct {
 mystr: String
}
```

--- a/crates/backend/src/codegen.rs
+++ b/crates/backend/src/codegen.rs
@@ -165,7 +165,7 @@ impl ToTokens for ast::Struct {
                     let ptr = js as *mut WasmRefCell<#name>;
                     assert_not_null(ptr);
                     let js = Box::from_raw(ptr);
-                    js.borrow_mut(); // make sure no one's borrowing
+                    (*js).borrow_mut(); // make sure no one's borrowing
                     js.into_inner()
                 }
             }

--- a/tests/wasm/classes.rs
+++ b/tests/wasm/classes.rs
@@ -1,3 +1,6 @@
+#[allow(unused_imports)] // test for #919
+use std::borrow::BorrowMut;
+
 use wasm_bindgen::prelude::*;
 use wasm_bindgen_test::*;
 

